spec/did.md

```diff
@@ -0,0 +1,80 @@
+# W3C Decentralized Identifiers v1.0
+
+The [DID Core data model](https://www.w3.org/TR/did-core) provides optionality for many of its properties, as it is designed as an [abstract data model](https://www.w3.org/TR/did-core/#representations). This means that many properties can be represented in different ways while still being spec conformant, properties can take on multiple concrete types (i.e. sometimes a string, sometimes an array), and documents can be extended to include additional properties either through the [DID Specification Registry](https://www.w3.org/TR/did-spec-registries/) or via usage of [JSON-LD](https://www.w3.org/TR/json-ld11/).
+
+This optionality can be difficult to implement consistently across languages. As a consequence, this specification defines a strict subset of the DID Core v1.0 data model represented by the following table. As a utility JSON Schemas for [DID Documents](did-document.json), [DID Resolution Metadata](did-resolution.json), and [DID Document Metadata](did-metadata.json) are provided to aid in the validation of conformant documents.
+
+## DID Document Data Model
+
+Following from [this data model](https://www.w3.org/TR/did-core/#core-properties).
+
+| Property      | JSON Representation | Required | Notes          |
+| ------------- | ------------------- | -------- | -------------- |
+| `id`          | String              | Yes      | Must be a URI. |
+| `@context`    | Array of strings    | No       | Depends on the DID method. |
+| `controller`  | Array of strings    | No       | Depends on the DID method. Strings must be URIs. |
+| `alsoKnownAs`        | Array of strings | No   | Depends on the DID method. Strings must be URIs. |
+| `verificationMethod` | Array of [Verification Methods](#verification-method-data-model) | Yes | There must be at least one Verification Method in each DID Document. |
```

Comment:
i _want to say_ that there's a scenario in building did dht diddocument where i know the id of the verificationMethod (which makes their way into one of 5 array of strings below), but not the verif method itself... but not 100% sure. 

either way, requiring this field to be present is a departure from the spec, yea?

Reply:
yes...have an upcoming dht pr to make that bit clearer with required relationships

and yes this is a departure from the spec

without a key I don't see the point in a DID Doc (feel free to convince me otherwise)

Reply:
this being a departure from the spec is a-ok with me! 

i did think that bit of including the id but not linking it back to an actual vm was weird, so i'm on board with this.

also side note, i am also probably going to be refactoring dht stuff in kt, so if you have a kt pr in flight we should coordinate

Reply:
Perhaps we've yet to encounter a practical scenario, but I agree with the above and don't see much utility for any use cases we're contemplating to have a DID Document that lacks at least one `verificationMethod` entry.

Additionally, I've personally not run into a case where "embedded" verification method usage in a verification relationship property is necessary.  In other words, a scenario where the only solution is to include a verification method embedded in a verification relationship property and NOT within the `verificationMethods` array.  I'd imagine the W3C spec contributors had some reason for including this in the DID Core spec, but I've come up blank in trying to identify the motivation? 

Seems like a question we might ask later -- but only to be prepared for a future Jeopardy round where 'Obscure DID Chronicles' is the surprise category.

Reply:
+1 to all of it